packages/backend/src/config/app-env-vars/m365.ts

```diff
@@ -0,0 +1,104 @@
+import appConfig from '../app'
+
+if (!appConfig) {
+  throw new Error('Cyclic import of appConfig from app-env-vars')
+}
```

Comment:
can i check is this just a safety precaution, like would this happen by accident or something?

Reply:
Yes, this is just in case. Should not happen